pytorch_lightning/accelerators/cpu.py

```diff
@@ -48,3 +53,27 @@ def auto_device_count() -> int:
     def is_available() -> bool:
         """CPU is always available for execution."""
         return True
+
+
+class CPUDeviceStatsEnum(LightningEnum):
+    """Enum for CPU device stats."""
+
+    CPU_VM_PERCENT = "cpu_vm_percent"
+    CPU_PERCENT = "cpu_percent"
+    CPU_SWAP_PERCENT = "cpu_swap_percent"
+
```

Comment:
i prefer defining these as constants within the file if they're used outside the module instead of defining an enum